tools/flaky-test-reporter/main.go

```diff
@@ -57,13 +58,21 @@ func main() {
 	if nil != err {
 		log.Fatalf("Failed removing local artifacts directory: %v", err)
 	}
+	var jobErrs []error
 	for repoName, jobList := range jobConfigs {
 		for _, jc := range jobList {
 			jc.Repo = repoName
 			log.Printf("collecting results for job '%s' in repo '%s'\n", jc.Name, jc.Repo)
 			rd, err := collectTestResultsForRepo(jc)
 			if nil != err {
-				log.Fatalf("Error collecting results for job '%s' in repo '%s': %v", jc.Name, jc.Repo, err)
+				errMsg := fmt.Sprintf("WARNING: failed collecting results, skipping '%s' in repo '%s': %v", jc.Name, jc.Repo, err)
```

Comment:
Instead of string, can we just make it an err?
```
e := fmt.Errorf("WARNING...")
log.Printf("%v", e)
err = append(err, e)
```

Reply:
good idea, done